go/azure_identity_credential_adapter.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The convention is to place the source files in a directory with the same name as the package.